barretenberg/cpp/bootstrap.sh

```diff
@@ -31,8 +31,11 @@ fi
 # Download ignition transcripts.
 (cd ./srs_db && ./download_ignition.sh 0)
 
+# Install wasi-sdk.
+./scripts/install-wasi-sdk.sh
```

Comment:
Do we want this after trying to use cache?

Reply:
It should only run once, and i think it makes sense to install it even if we take the output artefacts from the cache.

Reply:
Although you're making me second guess myself :)